release_scripts/Step2_Offline_PackageSigning.py

```diff
@@ -226,6 +226,105 @@ def getDstPath(pkgName, suffixStr='FileSuffix'):
 
 ################################################################################
 ################################################################################
+# Now verify multisig signatures before proceeding
+sys.path.append('/usr/lib/armory')
+from armoryengine.ALL import ADDRBYTE
+from jasvet import readSigBlock, verifySignature
+import urllib2
+import json
+import yaml # for assert file
+import deb822 # for buildinfo file
+
+# require valid signatures from threshold number out of total signers
+signers = ['ID/address', 'ID/address', 'ID/address', 'ID/address', 'ID/address']
+# Test signers that Joseph used
+#signers = ['LQmcPbkT/16veukrL1HicqY3VTN6qr2CgCtHVWCfNqt',
+#        'CeL6h2HD/19a5u6SMbGcj78Z78eXgRAUwBCZfiUW7ge',
+#        '14ra4Vd1/1Atcipo4vzCArFtxpbL6iLWbWZ3TFyotzK']
+threshold = 2
+
+for pkgName, pkgInfo in masterPkgList.iteritems():
+   print ''
+   print 'Verifying signatures for %s' % pkgName
+   hashDict = {}
+   sigCnt = 0
+   for signer in signers:
+      hashDict[signer] = []
+      print ''
+      print 'Verifying signature for signer %s' % signer
+      sigFile = os.path.join(inDir, 'armoryreproduciblesigs',
+              topVerStr, pkgInfo['OSNameLink'], signer,
+              'bitcoin-armory-%s-build%s.sig' % (pkgInfo['GitianName'],
+                  pkgInfo['GitianExt']))
+      print 'Accessing signature in file %s' % sigFile
+      try:
+         signature = open(sigFile).read()
+      except:
+         print 'Signature file does not exist'
+         continue
+      sig, msg = readSigBlock(signature)
+      addrB58 = verifySignature(sig, msg, 'v1', ord(ADDRBYTE))
+      if addrB58 == signer.split('/')[1]:
+         print 'Signature verified successfully for %s' % signer
+         # Remove "- " PGP message escaping
+         lines = msg.split('\n')
+         for idx, line in enumerate(lines):
+            if line[0:2] == '- ':
+               lines[idx] = line[2:]
+         msg = '\n'.join(lines)
+         
+         # Gitian YAML assert file parsing for hash
+         if pkgInfo['GitianExt'] == '.assert':
+            yaml.add_constructor(u'!omap',
+                    yaml.constructor.SafeConstructor.construct_yaml_omap)
+            msgYaml = yaml.load(msg)
+            msgHashes = msgYaml[0][1].split('\n')[:-1]
+            for msgHash in msgHashes:
+               hashDict[signer].append(msgHash.split('  ')[0])
```

Comment:
Yes, it is intentional. It seems like the assert file uses the format that sha256sum can output/can verify, which is hash-space-space-filename.